control-plane/grpc/proto/v1/replica/replica.proto

```diff
@@ -59,6 +61,16 @@ enum ReplicaStatus {
   Faulted = 3;
 }
 
+// Kind of the Replica.
+enum ReplicaKind {
+  // A regular data-replica.
+  Regular = 0;
+  // A replica's snapshot, which is a replica.
```

Comment:
`A replica which is created as a snapshot of some replica.` ?

Reply:
Tweaked it a bit:
    /// A replica which is another replica's snapshot.
    /// A replica which is a clone of another replica's snapshot.